README.md

```diff
@@ -1,40 +1,66 @@
 # Email alert frontend
 
-## Purpose
+A frontend for creating and managing email subscriptions.
 
-![Email Alert Frontend](docs/email-alert-frontend.png)
+## Overview
 
-A consistent frontend for displaying email alert signup pages.
+### Signup
 
-## Live examples
+This app provides two routes for signing up to email:
 
-- [gov.uk/government/policies/academies-and-free-schools/email-signup](https://www.gov.uk/government/policies/academies-and-free-schools/email-signup)
+- A legacy `/:base_path/email-signup` route [[example](https://www.gov.uk/foreign-travel-advice/canada/email-signup)].
+
+Each of these routes corresponds to a content item with an [`email_alert_signup` schema](https://github.com/alphagov/govuk-content-schemas/blob/master/formats/email_alert_signup.jsonnet). At the time of writing, `/foreign-travel-advice/*` and `/service-manual/*` still use this route.
+
+- A `/email-signup?link=/:base_path` route [[example](https://www.gov.uk/email-signup/?topic=/money)].
+
+This route supports signup to several types of content. It was [originally specific to the taxonomy](https://github.com/alphagov/email-alert-frontend/pull/33) (despite the generic name), and then [got re-purposed for other document types](https://github.com/alphagov/email-alert-frontend/pull/451).
+
+### Double Opt-In
+
+In order to verify the email for a new subscription:
+
+- We [send a verification email using email-alert-api](https://github.com/alphagov/email-alert-frontend/pull/629)
+- The email [contains a link with a unique token for the subscription](https://github.com/alphagov/email-alert-api/pull/1051)\*
+- Clicking on the link [completes the signup process](https://github.com/alphagov/email-alert-frontend/pull/615)
+
+\* Note that [the token strategy has changed over time](https://github.com/alphagov/email-alert-api/pull/1083).
+
+### Manage
+
+This allows the user to list, modify and delete their subscriptions [[login](https://www.gov.uk/email/manage/authenticate)]. It uses a similar, but separate email/token process to authenticate a user, establishing a session for them to make their changes.
 
 ## Nomenclature
 
-- Email Alert Signup: a page displaying the title of what the User is signing up to
-- Tags: A key and array value pair which when are used by the [Email Alert API](https://github.com/alphagov/email-alert-api) to forward the user onto the signup URL
+### Tags and links
 
-## Running the test suite
+Uniquely define a list people can subscribe to. The criteria within are used to figure out whether an update is relevant subscribers in the list. Defined in [the docs for email-alert-api](https://github.com/alphagov/email-alert-api/blob/master/doc/matching-content-to-subscriber-lists.md).
 
 ```
-$ bundle exec rake
+tags: { topics: { any: ["business-tax/vat"] } }
 ```
 
-## Content Item
+```
+links: { topics: { any: ["1ddaaacb-0981-4abc-a532-a5111f2bea6b"] } }
+```
+
+### Tags and links (again!)
```

Comment:
Think this heading could be clearer as "Legacy tags and links"?

Reply:
Sure, updated 👍